src/styles/page.scss

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: initial;
+}
+
+.calendar {
+  display: flex;
+  justify-content: center;
+
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  cursor: pointer;
+
+  &__list {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $cell-gap;
+
+    font-size: 188%;
+
+    max-width: $width-calendar;
+    margin-top: 40px;
+    padding: 10px;
+  }
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $cell-size;
+    height: $cell-size;
+
+    border: 1px solid $border-color;
+    background-color: $background-color;
+    transition: 0.5s;
+
+    &:hover {
+      background-color: $background-color-hover;
+      transform: translate(0, -20px);
+    }
+
+    @for $cell-num from 1 through 31 {
+      &:nth-child(#{$cell-num})::before {
+        content: "#{$cell-num}";
+      }
+    }
+  }
+
+  @each $day, $num in $days-of-the-week {
+    &__day--#{$day}:first-child {
+      margin-left: $cell * $num;
+    }
+  }
```

Comment:
Intead of` calendar__day--sat` you have to create modifier class for `calendar ` class like `calendar--month-length--sat`